tools/mob_suite/mob_typer.xml

```diff
@@ -0,0 +1,72 @@
+<tool id="mob_typer" name="MOB-Typer" version="1.4.1">
+  <description>(Get the plasmid type and mobility given its sequence)</description>
+  <requirements>
+     <requirement type="package" version="1.4.1">mob_suite</requirement>
+     <!--requirement type="package" version="2.7.1">blast</requirement-->
+     <!--requirement type="package" version="2.0">mash</requirement-->
+     <!--requirement type="package" version="1.5.5">circlator</requirement-->
+  </requirements>   
+  <command detect_errors="exit_code">
+    ln -s ${input} ${input.name};
+    mob_typer  --num_threads 16 --infile "${input.name}" --evalue ${adv_param.evalue} --min_ident ${adv_param.min_ident} --min_cov ${adv_param.min_cov} --outdir ".";
+    ##/home/kbessonov/miniconda3/envs/general/bin/python /home/kbessonov/miniconda3/envs/general/bin/mob_typer  --num_threads 16 --infile "${input}" --outdir ".";
```

Comment:
Don't hardcode paths.
Have a look at this example: https://github.com/phac-nml/galaxy_tools/blob/a7add3a4ffc9275f78d4583aac1ef3281c0def21/tools/bam2mappingstats/bam2mappingstats.xml#L13